### ISSUE TITLE
pipe() not supported on dummy fs targets

### ISSUE THREAD

**Issue by damianloew** [phoenix-rtos/phoenix-rtos-project#684]
Because of the fact that [posixsrv](https://github.com/phoenix-rtos/phoenix-rtos-posixsrv/tree/5c8501120ffb3a2f4aee4f579988c04c80aa019d) is not enabled on targets with dummy fs it's not possible to use `pipe()` function properly, it returns `ENOSYS` errno. For now it's not planned to support this utility.

**Comment by rmikielis** [comment]
I think this is no longer standing, posixsrv/pipes working correctly on armv7r5f-zynqmp-qemu.

**Comment by nalajcie** [comment]
Yeah, depends on the target configuration - you can embed libposixsrv into the multi driver if the project requires it.
